helm/dagster/charts/dagster-user-deployments/templates/helpers/_helpers.tpl

```diff
@@ -129,6 +129,9 @@ DAGSTER_K8S_PIPELINE_RUN_ENV_CONFIGMAP: "{{ template "dagster.fullname" . }}-pip
     {{- if .resources }}
     resources: {{- .resources | toYaml | nindent 6 }}
     {{- end }}
+    {{- if .schedulerName }}
+    scheduler_name: {{ .schedulerName }}
+    {{- end }}
```

Comment:
this won't be set unless they explicitly override it right? the default case is for it to not be included?

Reply:
correct